modules/transformer-batch/src/main/resources/application.conf

```diff
@@ -25,6 +25,7 @@
     "tsv": [],
     "skip": []
```

Comment:
I've checked whether `skip` is working with shredding but it seems like it doesn't work as intended. It is only used in [here](https://github.com/snowplow/snowplow-rdb-loader/blob/741f7f114ad0ff1a62bd5b70068a76c65024eba4/modules/common/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/common/Common.scala#L78). According to this part, it doesn't skip the schema. It just causes to json shredding instead. Since it is like this for a long time, we can leave it as it is I guess. But, I've added the `skipSchemas` functionality for TSV transformation as well.

Reply:
> Can you think of any reason why somebody would use the existing skip feature, as it is currently implemented?

I don't think anyone is using it at the moment because it doesn't do what it advertised at the moment. Therefore, it should be okay to remove it I guess.

> If it is completely useless (I suspect it is!!) then we should remove it. And maybe your new feature should become the skip feature, instead of introducing a new option skipSchemas?

I thought about it but I couldn't be sure if it is good idea to add `skip` field to `formats` section for widerow transformations as well. In the historical context of TSV transformation, it can be understood why `skip`, `json` and `tsv` fields are added to `formats` section but since we want to add `skip` for all formats, I don't think `formats` section good choice for `skip` field. Therefore, I thought it would be better to add new `skipSchemas` section would be better. As backward compatibility, I agree it would be better to add `skip` to `formats` section but if we think `skip` isn't used at all, I guess it should be okay to just add new field like it is completely new feature. But, I don't have strong opinion on this one. Using existing `skip` field is okay as well I think.

Reply:
Oh I had not realised that `skip` is a sub-category of `formats`.  In this case I agree that top-level `skipSchemas` is better.

> I don't think formats section good choice for skip field.

Agree.